src/components/__tests__/EthAddress.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { simpleRender, fireEvent } from 'test-utils';
+
+import { truncate } from '@utils';
+
+import EthAddress, { Props } from '../EthAddress';
+
+const defaultProps: Props = {
+  address: '0x4bbeEB066eD09B7AEd07bF39EEe0460DFa261520',
+  truncate
+};
+
+function getComponent(props: Props) {
+  return simpleRender(<EthAddress {...props} />);
+}
+
+// @ts-ignore
+navigator.clipboard = { writeText: jest.fn() };
+
+describe('EthAddress', () => {
+  test('it shows the address untruncated', async () => {
+    const { getByText } = getComponent({ ...defaultProps, truncate: undefined });
```

Comment:
Since this is used consistently throughout the app with the same behaviour, it seems better for truncate to be a boolean.